src/sentry/tasks/beacon.py

```diff
@@ -14,33 +14,49 @@
 from sentry.tasks.base import instrumented_task
 from sentry.utils import json
 
-BEACON_URL = "https://sentry.io/remote/beacon/"
+#  BEACON_URL = "https://sentry.io/remote/beacon/"
+BEACON_URL = "https://dev.getsentry.net:8000/remote/beacon/"
 
 logger = logging.getLogger("beacon")
 
 
-@instrumented_task(name="sentry.tasks.send_beacon", queue="update")
-def send_beacon():
-    """
-    Send a Beacon to a remote server operated by the Sentry team.
-
-    See the documentation for more details.
-    """
+def get_install_id():
     from sentry import options
-    from sentry.models import Broadcast, Organization, Project, Team, User
 
     install_id = options.get("sentry:install-id")
     if not install_id:
         install_id = sha1(uuid4().bytes).hexdigest()
         logger.info("beacon.generated-install-id", extra={"install_id": install_id})
         options.set("sentry:install-id", install_id)
 
+    return install_id
+
+
+def should_skip_beacon(install_id):
     if not settings.SENTRY_BEACON:
         logger.info("beacon.skipped", extra={"install_id": install_id, "reason": "disabled"})
```

Comment:
I don't think we need the `install_id` here in this helper function. We can output the same log line without the install id as install id should never change for an installation.

We can move the `install_id` enriched log line to the main `send_beacon` method:
```py
logger.debug("beacon.sending", extra={"install_id": install_id})
```

Reply:
I'd prefer to keep the log messages as-is, I'm not sure of the context and the other log messages all include the `install_id`

Reply:
I would be very surprised if _anyone_ relies on these log messages. Remember, these are logged on individual self-hosted machines and very few people if any care about the beacon pings, let alone their install ids.